shard.yml

```diff
@@ -3,11 +3,13 @@ name: amber
 version: 0.6.4
 
 authors:
-  - Elias Perez <<eliasjpr@gmail.com>>
-  - Isaac Sloan <<isaac@isaacsloan.com>>
-  - Dru Jensen <<drujensen@gmail.com>>
-  - Nick Franken <<shnick@gmail.com>>
-  - Benoit de Chezelles <<benoit.dechezelles@gmail.com>>
+  - Dru Jensen <drujensen@gmail.com>
```

Comment:
I was thinking that instead of maintaining this list we can just add something more general and inclusive

`Amber Core Team and Contributors`